src/components/Graph/PrimaryChart/PrimaryChart.tsx

```diff
@@ -24,19 +25,34 @@ export interface PrimaryChartProps {
   data: HistoryData[]
   width: number
   height: number
-  margin?: { top: number; right: number; bottom: number; left: number }
+  margin: { top: number; right: number; bottom: number; left: number }
   color?: string
 }
 
 // accessors
 const getStockValue = (d: HistoryData) => d?.price || 0
 
+const verticalCrosshairStyle = {
+  stroke: colors.blue[500],
+  strokeWidth: 2,
+  opacity: 0.5,
+  strokeDasharray: '5,2',
+  pointerEvents: 'none',
+}
+
+const tooltipStyle = { zIndex: 10 } // render over swapper TokenButton component
+
+const accessors = {
+  xAccessor: (d: HistoryData) => d.date,
+  yAccessor: (d: HistoryData) => d.price,
+}
```

Comment:
We probably want to make a `useMemo` out of this guy and/or split into two `useCallback`s - since these are functions and react is using shallow comparison, every re-render of this would end up of `xAccessor` and `yAccessor` being referentially instable